iotlabcli/tests/main_parser_test.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+# This file is a part of IoT-LAB cli-tools
+# Copyright (C) 2015 INRIA (Contact: admin@iot-lab.info)
+# Contributor(s) : see AUTHORS file
+#
+# This software is governed by the CeCILL license under French law
+# and abiding by the rules of distribution of free software.  You can  use,
+# modify and/ or redistribute the software under the terms of the CeCILL
+# license as circulated by CEA, CNRS and INRIA at the following URL
+# http://www.cecill.info.
+#
+# As a counterpart to the access to the source code and  rights to copy,
+# modify and redistribute granted by the license, users are provided only
+# with a limited warranty  and the software's author,  the holder of the
+# economic rights,  and the successive licensors  have only  limited
+# liability.
+#
+# The fact that you are presently reading this means that you have had
+# knowledge of the CeCILL license and that you accept its terms.
+
+""" Test the iotlabcli.experiment_parser module """
+# pylint: disable=too-many-public-methods
+# pylint: disable=invalid-name
+
+import pytest
+
+import iotlabcli.parser.main as main_parser
+
+from .c23 import patch
+
+
+entrypoints = [
```

Comment:
The entrypoints could be directly in the parametrize decorator:
```python
@pytest.mark.parametrize('entry', ['auth', 'admin', 'experiment', 'node', 'profile', 'robot'])
```

Then I think there's no need for `# pylint: disable=invalid-name`

Reply:
ok. needed a tuple of tuple but this is ok. pushed a fixup

Reply:
Do you know why ? Because normally, you can just use a string for the parameter name ('entry', and a list of possible values as string or whatever)

Reply:
you're right. 'entry', then list of values. I had put ['entry']